internal/pkg/runtime/engine/apptainer/cleanup_linux.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/apptainer/apptainer/internal/pkg/util/priv"
 	"github.com/apptainer/apptainer/internal/pkg/util/starter"
 	"github.com/apptainer/apptainer/pkg/build/types"
+	apptainer "github.com/apptainer/apptainer/pkg/runtime/engine/apptainer/config"
```

Comment:
Most other source files in this directory call this `apptainerConfig`.  I see it is inconsistent in container_linux.c.  It should probably be changed there to `apptainerConfig` too.